libass/ass_fontselect.c

```diff
@@ -168,22 +168,34 @@ static void load_fonts_from_dir(ASS_Library *library, const char *dir)
     DIR *d = opendir(dir);
     if (!d)
         return;
+    size_t dirlen = strlen(dir);
```

Comment:
There already is a `bufsize`-variable in this method *(for the font file contets)*, so shorting to just `buf` and `len` would imho be confusing. `namemax`/`name_max` seems good. I can add underscores if you prefer *(ironically this will make the names longer)*, but the aforementioned `bufsize` also doesn't use an underscore so this may be inconsistent.